packages/mux-active-viewer-count/package.json

```diff
@@ -41,7 +41,7 @@
   "license": "MIT",
   "scripts": {
     "clean": "shx rm -rf dist/ && shx rm -rf src/polyfills",
-    "lint": "eslint src/ --ext .js,.jsx,.ts,.tsx",
+    "lint": "ESLINT_USE_FLAT_CONFIG=false eslint src/ --ext .js,.jsx,.ts,.tsx",
```

Comment:
NOTE: Upgraderd eslint and related to latest version for monorepo tooling dependency consistency. After upgrading, `ESLINT_USE_FLAT_CONFIG=false` was added due to https://github.com/eslint/eslint/issues/16531#issuecomment-1331258992